### PR TITLE
Update of the codeowners file

### PR DESCRIPTION
**Pull request by Greg-21**
I added the missing launcher translation files, but I commented them out for now to prevent them from taking effect.

I have also a small question: are the doubled "*" necessary? Isn't one enough?

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
+
+# assets/launcher/lang/sklmessages_cs_CZ.**
 assets/launcher/lang/sklmessages_en_US.** @skmedix
 assets/launcher/lang/sklmessages_eo.** @Waterrail
 assets/launcher/lang/sklmessages_es_ES.** @atesin
 # assets/launcher/lang/sklmessages_fr_FR.**
-assets/launcher/lang/sklmessages_hu_HU.** @Gbr22 
+assets/launcher/lang/sklmessages_hu_HU.** @Gbr22
 assets/launcher/lang/sklmessages_nb_NO.** @SirLexPLAY
 assets/launcher/lang/sklmessages_pl_PL.** @Waterrail @Greg-21 @Indexer
```
Comment (skmedix) [inline .github/CODEOWNERS]: I would shrink PL CODEOWNERS to one, as GH only supports one owner per folder/file.

Comment (skmedix) [reply .github/CODEOWNERS]: Yeah, only the first one is getting review request. Probably a bug, but existing since

Comment (skmedix) [reply .github/CODEOWNERS]: I don't think so

Comment (Greg-21) [reply .github/CODEOWNERS]: So I'll remove also them.

Comment (Greg-21) [reply .github/CODEOWNERS]: So I'm going to remove also them.

**Pull request merged**